### ISSUE TITLE
Could not find package git-commit

### ISSUE THREAD

**Issue by jsjoeio** [hlissner/doom-emacs#5116]
### 

- [X] I have searched [the FAQ](https://doomemacs.org/docs/faq.org) and [the documentation](https://doomemacs.org/docs/getting_started.org) for solutions.
- [X] I have searched [the issue tracker](https://github.com/hlissner/doom-emacs/issues) for similar issues (the closed issues too).
- [X] I have searched [the Discourse](https://doomemacs.org/d/docs) for any errors and solutions.
- [X] I can reproduce my issue on [the latest commit](https://github.com/hlissner/doom-emacs/commit) of Doom Emacs.
- [X] I have read [How to Debug Issues](https://doomemacs.org/d/how2debug) and found no solution.

### What did you expect to happen?

Doom to install correctly.

### What actually happened?

Installation fails due to unexpected error. Can't find package `git-commit`.

![58439B0D-3385-4DE3-B7FB-862F43D2EC5D](https://user-images.githubusercontent.com/3806031/120124412-45b2ba80-c169-11eb-81d4-2c8abfb8ba18.png)

See full output of [doom.error.log](https://pastebin.com/7jnMgikf).

### Describe your attempts to resolve the issue

I tried reinstalling Doom.

I tried running `doom sync`. 

I tried looking for previously closed issues but couldn't find anything.

### Steps to reproduce

1. Download [ISH app](https://ish.app/) on iPad
2. Upgrade Alpine Linux to 3.13
3. Download emacs `apk add emacs`
4. Follow Doom installation instructions
5. Observe error

### System Information

https://pastebin.com/trCMLG3U

**Comment by hlissner** [comment]
`git-commit` is available on MELPA, therefore your local recipe repos are likely out of date or failed to clone correctly. Do the following:

```sh
rm -rf ~/.emacs.d/.local/straight
doom sync
```

As this is a local issue, I will close this, but feel free to follow up here.

**Comment by jsjoeio** [comment]
Thank you for the quick reply! 

It looks like both `doom sync` and `doom build` are hanging

![image](https://user-images.githubusercontent.com/3806031/120135902-165e7680-c186-11eb-82a8-baf44a491670.png)

I found [this issue](https://github.com/hlissner/doom-emacs/issues/3655#issuecomment-725969086) with some suggestions.

Honestly, I may just have to purge everything and restart. I'm guessing I made a silly mistake somewhere along the process.
